<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>650</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>650</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>45</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>基于嵌入式Qt的多功能电子台历</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setClockButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置闹钟</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>100</height>
     </size>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>380</width>
      <height>210</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>210</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>210</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>214</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="text">
          <string>温度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="text">
          <string>湿度:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="temperatureLabel">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="text">
          <string>999</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="humidityLabel">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="text">
          <string>999</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="text">
          <string>℃</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="text">
          <string>%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>380</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2022-12-31</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="syncButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>网络校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手动调整</string>
    </property>
   </widget>
   <widget class="QLabel" name="copyrightLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
